code/modules/mob/living/simple_animal/animals/chungus.dm

```diff
@@ -0,0 +1,155 @@
+/mob/living/simple_animal/hostile/chungus
+	name = "big chungus"
+	desc = "There's a chungus amongus."
```

Comment:
Hooooly shit, hahaha.

Reply:
Addition by somebody else, wanted me to add it for them.

Reply:
Got a good laugh reading through the diff. Nicely done